addons/website_quote/views/website_quotation.xml

```diff
@@ -253,12 +255,13 @@
                       </div>
                       <a id="introduction"/>
                       <h1 class="page-header mt16">
-                          <span t-if="quotation.state in ('draft','sent','cancel')">Your Quotation</span>
-                          <span t-if="quotation.state not in ('draft','sent','cancel')">Your Order</span>
-                          <em t-esc="quotation.name"/>
-                          <small t-field="quotation.state"/>
+                          <span t-if="quotation.state in ('draft','sent','cancel')" class="quote-menu">Your Quotation</span>
+                          <span t-if="quotation.state not in ('draft','sent','cancel')" class="quote-menu">Your Order</span>
+                          <em t-esc="quotation.name" class="quote-menu"/>
+                          <small t-field="quotation.state" />
                           <div groups="base.group_website_publisher" t-ignore="true" class="pull-right css_editable_mode_hidden">
-                              <a class="btn btn-info hidden-print" t-att-href="'/web#return_label=Website&amp;model=%s&amp;id=%s&amp;action=%s&amp;view_type=form' % (quotation._name, quotation.id, action)">Update Quote</a>
+                              <a class="btn btn-info hidden-print" t-att-href="'/web#return_label=Website&amp;model=%s&amp;id=%s&amp;action=%s&amp;view_type=form' % (quotation._name, quotation.id, action)" t-if="quotation.state not in ('manual')">Update Quote</a>
+                              <a class="btn btn-info hidden-print" t-att-href="'/web#return_label=Website&amp;model=%s&amp;id=%s&amp;action=%s&amp;view_type=form' % (quotation._name, quotation.id, action)" t-if="quotation.state in ('manual')">Back to Sale Order</a>
```

Comment:
Put the `page-header` class on a new div containing the `h1` tag and the div with the buttons.